docs/manual/advanced-features.tex

```diff
@@ -1006,6 +1006,17 @@
 the method has no side effects, so calling it does not invalidate any
 dataflow facts.
 
+The \refqualclass{dataflow/qual}{SideEffectsOnly} annotation
+specifies the superset of expressions that the method might side effect.
```

Comment:
"the superset" is confusing; I'm not sure what it is referring to.  I think you mean "a superset of the"